config/application.rb

```diff
@@ -26,5 +26,6 @@ class Application < Rails::Application
     config.site_image = "https://circuitverse.org/img/circuitverse2.svg"
     config.site_description = "Explore Digital circuits online with CircuitVerse. With our easy to use simulator interface, you will be building circuits in no time."
     config.slack_url = "https://join.slack.com/t/circuitverse-team/shared_invite/enQtNjc4MzcyNDE5OTA3LTdjYTM5NjFiZWZlZGI2MmU1MmYzYzczNmZlZDg5MjYxYmQ4ODRjMjQxM2UyMWI5ODUzODQzMDU2ZDEzNjI4NmE"
+    config.assets.paths << Rails.root.join("app", "assets", "fonts")
```

Comment:
Is this needed?

Reply:
this line add's the new created folder 'fonts' to the asset path in config/application.rb (it is always better to do that)
but things work correctly without it also ,If you wish I will remove it

Reply:
Yes, I appreciate that,  if we implement fonts  through vendor folder then we might need extra such line
```
app.config.assets.paths << Rails.root.join('vendor', 'assets', 'fonts')  
```
hence removed it ...
also  possible config I think we can add in assets.rb is just
```
app.config.assets.precompile << /\.(?:svg|eot|woff|ttf)$/
```
which is already present I, if any additional configs required then let me know